src/tool/cppwinrt/cppwinrt/main.cpp

```diff
@@ -35,6 +36,8 @@ namespace xlang
         { "filter" }, // One or more prefixes to include in input (same as -include)
         { "license", 0, 0 }, // Generate license comment
         { "brackets", 0, 0 }, // Use angle brackets for #includes (defaults to quotes)
+        { "fastabi", 0, 0 }, // Enable support for the Fast ABI
+        { "tests", 0, 0 }, // Generate internal test files
         { "ignore_velocity", 0, 0 }, // Ignore feature staging metadata and always include implementations
```

Comment:
no way to isolate this from the production exe?

Reply:
Yes, I've been thinking about eliding this somehow.